### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by pavlenex**
Continuation of #383
This creates templates for certain issues:
- Bug report
- Content request
- Image request

The other templates have to be added separately, in config.yml as described [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser). 

This PR aims to better direct people on what to submit were and provide them with quick templates they can fill in so we can have enough information without having to go back and forth. It also auto-assigns labels (copy, design, bug).

In the next PR I'll add:
- Request a feature (leading to discussion feature-request category, so that each feature can be discussed by community/contributors before it's implemented)
- :question:Post a question --> Discussion
- :speech_balloon: Community Chat --> Slack

This separation also aims to separate feature requests from actual issues. 
Issues aim to be actionable tasks (to-do) where as feature request are discussions that should be discussed and get a consensus on before they become the issue,

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---content-request.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+**Suggestions**
+_If you have a concrete suggestion on how to change/add the content, write it up here_
+
+**Additional context**
+
+_Add any other context about the request here._
+
+**Content guidelines**
+If you're a contributor looking to tackle the issue, you can [preview our illustration guidelines](https://bitcoin.design/guide/contribute/content-guidelines/).
```
Comment (johnsBeharry) [inline .github/ISSUE_TEMPLATE/---content-request.md:28]:
```suggestion
If you're a contributor looking to tackle the issue, you can [preview our content guidelines](https://bitcoin.design/guide/contribute/content-guidelines/).
```

**Review by johnsBeharry — CHANGES_REQUESTED** [review]

**Comment by pavlenex** [comment]
@GBKS I don't think so, perhaps we can try to do it in config.yml later on.

**Review by johnsBeharry — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/---image-request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Hello, I would like to request a visual for the ...
+
+**Link to the page**
+
+_Please add a link to the page you're requesting a visual for._
+
+**Visual Type**
+
+What kind of visual are you requesting? (Header image, content image, video, prototype, etc.)
```
Comment (Bosch-0) [inline .github/ISSUE_TEMPLATE/---image-request.md:20]: Prototypes and static visuals could prob each be their own template as they are quite different and will likely be contributed by people with different skills sets (images more graphic designers, prototypes more UI/UX focused designers).

Comment (pavlenex) [reply .github/ISSUE_TEMPLATE/---image-request.md:20]: Too many templates = confusion. Let's keep it simple. The request template will probably be the same regardless.

Comment (Bosch-0) [reply .github/ISSUE_TEMPLATE/---image-request.md:20]: How's about call this media request which shows it encompasses opening an issue for any kind of media (prototype, image, video etc.)

Comment (pavlenex) [reply .github/ISSUE_TEMPLATE/---image-request.md:20]: I think the `image` will be 90% of the requests tbh, let's see if it confuses people and if it does, we can, later on, add it as a `media` template. Let's first see how people use it.

**Pull request merged**